frontend/src/app/features/work-packages/components/wp-tabs/services/wp-tabs/wp-files-count.function.ts

```diff
@@ -43,7 +43,7 @@ export function workPackageFilesCount(
 
   return combineLatest(
     [
-      attachmentService.collection(workPackage.$links.attachments.href || ''),
+      workPackage.$links.attachments ? attachmentService.collection(workPackage.$links.attachments.href || '') : new BehaviorSubject([]),
```

Comment:
🟡 why a BehaviorSubject? And, please lets extract that into a variable to keep the combineLatest readable.

Reply:
Extracted.
I assume I can't use plain empty array with combineLatest and I need an observable to emit it. So, I used BehaviorSubject to do it. I am not proficient with RxJs and there can be complete misunderstanding from my side. So, feel free to correct me.

Reply:
usually you use `of([])` to create a simple observable of an empty array.